src/main/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -20,20 +22,20 @@ public SelectTestFigureOptionListener(DriverManager driverManager, TestShape sha
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
+		Job2dDriver driver = driverManager.getCurrentDriver();
+		ComplexCommandFactory factory = new ComplexCommandFactory();
 		switch (this.testShape) {
 			case FIGURE1:
-				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+				FiguresJoe.figureScript1(driver);
 				break;
 			case FIGURE2:
-				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+				FiguresJoe.figureScript2(driver);
 				break;
-
 			case RECTANGLE:
-				ComplexCommandFactory.getRectangle().execute(driverManager.getCurrentDriver());
+				factory.drawRectangle(100,100,100,100).execute(driver);
 				break;
-
 			case STAR:
-				ComplexCommandFactory.getStar().execute(driverManager.getCurrentDriver());
+				factory.drawStar().execute(driver);
 				break;
 		}
```

Comment:
Command tests should load commands to DriversCommandManager (setCurrentCommand), like topSecretCommand. Later command can be executed using Run Command option in menu.